rename.ml

```diff
@@ -0,0 +1,74 @@
+open Instr
+
+let fresh_var instrs var =
+  let cand i = var ^ (string_of_int i) in
```

Comment:
[gasche] I would propose that `cand "x" 0` be `"x"` instead of `"x0"` to have the nice sequence x,x1,x2...